package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "d3-dag",
-  "version": "0.4.5",
+  "version": "0.4.6",
```

Comment:
Don't worry about changing the version, I'll change it before I publish in it's own version bump commit.